### ISSUE TITLE
Can you show the Global TouchBar settings only when pressing the BTT button on the control strip?

### ISSUE THREAD

**Issue by PushFoward** [folivoraAI/BetterTouchTool#1195]
As it stands, I have the Global Settings on my TouchBar set up to show quite a lot of functions that are of immense help when listening to music, but irrelevant when not.

I want them to be in the Global settings, so that I can access them from any app, but I dislike that they take priority anytime my Macbook comes out of Sleep mode. Is there a way to have my custom global settings pop up only if I press the BTT icon on the control strip? So that I see the global settings only when I want to access them specifically?

**Comment by kamykaze** [comment]
Hi, I ran into a similar situation as yours when I first started configuring BTT with touchbar. 

I think the challenge with your solution is that whichever app you're using to play music will probably have an icon in the control strip itself that will take over BTT's icon.

My approach to having quick access to certain "global" functions without always showing them is this:

- I don't have have any touchbar buttons set globally unless they use a modifier key (that way I keep the app specific buttons which is sometimes useful)
- holding CTRL: media specific buttons, folder shortcuts to Home, Applications, etc., shortcut to settings (BTT, system preferences, karabiner, etc.)
- holding ALT: app specific functions (eg: Chrome user profiles, incognito, etc.)
- holding CMD: app switcher, window resizing buttons, etc.
- holding Fn: expanded system control strip (not handled by BTT, it's a setting in Keyboard preferences)

In other words, by holding one modifier key, I can access any touchbar function I need. Without any keys, the touchbar behaves with its normal functionality

Note that I didn't use Shift, or any combination of two modifiers, so you could have a lot more "sets" of touchbar functionality if you want to.
